run.sh

```diff
@@ -1,9 +1,5 @@
 #!/bin/bash
+echo "Building Docker container"
```

Comment:
Should this be `running` instead of `building`? I don't see any build commands anymore in this file.

```suggestion
echo "Running Docker container"
```

Reply:
Ah yes that should be running, I had previously left that over from the script would build the container. I tend to associate composing with building.